config/config.go

```diff
@@ -1162,6 +1167,22 @@ func LoadFile(filename string) (*Config, error) {
 	return cfg, nil
 }
 
+var envVarRegex = regexp.MustCompile(`\${([a-zA-Z_][a-zA-Z0-9_]*)}`)
```

Comment:
why such a regexps and not just `\${(.*)}` ?

Reply:
This is based on the POSIX accepted environment variables, see: https://stackoverflow.com/questions/2821043/allowed-characters-in-linux-environment-variable-names